Makefile

```diff
@@ -166,8 +166,9 @@ test/addons/.buildstamp: config.gypi \
 	# Cannot use $(wildcard test/addons/*/) here, it's evaluated before
 	# embedded addons have been generated from the documentation.
 	for dirname in test/addons/*/; do \
-		echo "\nRunning addons test $$PWD/$$dirname" ; \
-		$(NODE) deps/npm/node_modules/node-gyp/bin/node-gyp --loglevel=$(LOGLEVEL) rebuild \
+		echo "\nBuilding addon $$PWD/$$dirname" ; \
+		MAKEFLAGS="-j1" $(NODE) deps/npm/node_modules/node-gyp/bin/node-gyp \
```

Comment:
`env MAKEFLAGS="-j1" ...`?  I believe what you have now is a bash-ism, not a posix-ism.